lib/src/widgets/help_collection/first_help_widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+
+class FirstHelpWidget extends StatelessWidget {
+  ///Constructor
+  const FirstHelpWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return HelpWidget(
+      mainCardTitle: 'Stop War! You can help!',
+      options: defaultOptionsList,
+      optionsCardBuilder: (options, onClose) {
+        return CardRounded(
+          key: const ValueKey(2),
+          child: LinksCardWidget(
+            options: options,
+            onHidePressed: onClose,
+          ),
+        );
+      },
+      mainCardBuilder: (flag, title, desc, onDetails, onClose) {
+        return CardRounded(
+          key: const ValueKey(1),
```

Comment:
Why this key here?

Reply:
It is just for AnimatedSwitcher, he is have problems with recognizing widgets, if you goona have 2 CardRounded widgets and trying to switch between them, AnimatedSwitcher will not recognize, that is 2 different widgets.